### ISSUE TITLE
Render the client on the server isometrically.

### ISSUE THREAD

**Issue by mythmon** [mythmon/edwin#12]
Eventually, we should render the React client on the server side for the first page load, and ship the rendered HTML to the browser. This makes faster page loads, and enables clients that don't allow JS. This depends on #10.

This is a stretch goal, and is really just to get practice with isomorphic react apps.

**Comment by mythmon** [comment]
On second thought, I'm going to close this. It would be nice in the future, but it doesn't need to stick on the todo list until then.
